### ISSUE TITLE
attempt to index a nil value ( field 'metadata')

### ISSUE THREAD

**Issue by marekklein** [jperon/lyluatex#302]
Hello, I have a problem when I try to generate a score. Last lines of log are here:
`*geometry* driver: auto-detecting
*geometry* detected driver: luatex
(/usr/share/texlive/texmf-dist/tex/latex/pdflscape/pdflscape.sty
(/usr/share/texlive/texmf-dist/tex/latex/graphics/lscape.sty))/usr/share/texliv
e/texmf-dist/scripts/lyluatex/lyluatex.lua:1335: attempt to index a nil value (
field 'metadata')
stack traceback:
	/usr/share/texlive/texmf-dist/scripts/lyluatex/lyluatex.lua:1335: in function 
'lyluatex.lua.get_font_family'
	[\directlua]:1: in main chunk.
\ly@currentfonts ..._font_family(font.current()))}
                                                  \rmfamily \edef \rmfamilyi...

l.37 \includely[]{horvath-missa_lux_et_origo1.ly}
`
Can you help me?
[book.log](https://github.com/jperon/lyluatex/files/10783034/book.log)

**Comment by jperon** [comment]
Would you please have a MWE, so that I can look into it ? Otherwise, if you don’t mind sharing your document, you could <a href="mailto:cataclop@hotmail.com?subject=LyLuaTeX #302&body=Here is a document illustrating issure #302.">send it to me</a>.

**Comment by jperon** [comment]
Thank you for the document. The problem comes from those lines, that are obsolete:

```tex
\usepackage[T1]{fontenc}
\usepackage[utf8]{luainputenc}
```

You should replace them by:

```tex
\usepackage{fontspec}
```

You may also call a package to define the global font: under `lualatex`, they depend on `fontspec`, so no need to load it explicitly. Moreover, if adding the `pass-fonts` option to `lyluatex`, you would get a coherent font for you document:

```tex
\usepackage{ebgaramond}  % You may replace ebgaramond by libertine, for example.
\usepackage[pass-fonts]{lyluatex}
```

**Comment by marekklein** [comment]
Thank you.
I get other errors now and the compiling hangs after the last line of the
attached log...

**Comment by jperon** [comment]
I fear the log didn’t get attached.
________________________________
De : Marek Klein ***@***.***>
Envoyé : mercredi 22 février 2023 07:08
À : jperon/lyluatex ***@***.***>
Cc : jperon ***@***.***>; Comment ***@***.***>
Objet : Re: [jperon/lyluatex] attempt to index a nil value ( field 'metadata') (Issue #302)


Thank you.
I get other errors now and the compiling hangs after the last line of the
attached log...

**Comment by jperon** [comment]
Fixed in 1.1.
